draft/6/sign-lgn-notation/index.md

```diff
@@ -139,8 +132,8 @@ The following identities hold:
 
 The following identities hold:
 
-- `[A: B] == A B (A)'`
-- `[A, B] == A B (A)' (B)'`
+- `[A: B] == A B A'`
```

Comment:
There's a small issue with this, in that this is unsafe at a syntactic level.

It would be possible to interpret this as:

    [R U: F] == R U F R U'

I think this can be fixed by specifying that e.g. `A == (A)`, but that also requires specifying that inverses cancel, etc. That's probably best not handled at the syntax level.

Reply:
Hm, ok, did not think about such misinterpretations...
I understand